samples/nrf9160/modem_shell/src/location/location_shell.c

```diff
@@ -223,8 +223,8 @@ void location_ctrl_event_handler(const struct location_event_data *event_data)
 			"  used method: %s (%d)",
 			location_method_str(event_data->location.method),
 			event_data->location.method);
-		mosh_print("  latitude: %.06f", event_data->location.latitude);
-		mosh_print("  longitude: %.06f", event_data->location.longitude);
+		mosh_print("  latitude: %.08f", event_data->location.latitude);
+		mosh_print("  longitude: %.08f", event_data->location.longitude);
```

Comment:
General comment about this. Changing %.06f to %.08f is done just because nRF Cloud seems to show the position with 8 decimals. So this is not optimal if cloud changes one day to be even more accurate.